jmeter_test/Dockerfile

```diff
@@ -0,0 +1,41 @@
+FROM alpine:3.10
+MAINTAINER tofarr tofarrell@codeforamerica.org
+
+# Install packages
+#RUN apk update && \
+#    apk add ca-certificates wget python python-dev py-pip && \
+#    update-ca-certificates && \
+#    pip install --upgrade --user awscli
```

Comment:
can we delete this since it's commented out?

Reply:
Done!